src/components/intro/intro.jsx

```diff
@@ -15,12 +15,20 @@ class Intro extends React.Component {
     constructor (props) {
         super(props);
         bindAll(this, [
+            'handleClickRegistration',
             'handleShowVideo'
         ]);
         this.state = {
             videoOpen: false
         };
     }
+    handleClickRegistration (event) {
```

Comment:
Is there a way to share this and navigateToRegistration between navigation.js and here? It'd be nice if the code were shared so we don't run into this again.

Reply:
Hm, I think you're right, that would be better.

I rewrote the whole thing to refactor that logic into the reducer. take a look.